Array Processing Tool/array_processing_tool.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Array Processing Tool</title>
+        <link rel="stylesheet" href="main.css" type="text/css">
+    </head>
+    <body>
+        <section>
+            <h1>Sub sum</h1>
+            <form id="form_subSum">
+                <span>Enter array: </span>
+                <input id="arr_subSum" type="text" list="arrays">
+                <datalist id="arrays">
+                    <option>-1, 2, 3, -9</option>
+                    <option>2, -1, 2, 3, -9</option>
+                    <option>-1, 2, 3, -9, 11</option>
+                    <option>-2, -1, 1, 2</option>
+                    <option>100, -9, 2, -3, 5</option>
+                    <option>1, 2, 3</option>
+                    <option>-1, -2, -3</option>
+                </datalist>
+                <input class="btn" type="submit" value="Get sum">
+            </form>
+            <p id="answer_subSum">Answer will be <br> here...</p>
+        </section>
+
+        <section>
+            <h1>Search</h1>
+            <form id="form_search">
+                <span>Enter array: </span>
+                <input id="arr_search" type="text" list="arrays">
```

Comment:
![image](https://user-images.githubusercontent.com/2150198/93859891-e42b7980-fcc6-11ea-9061-7e5887f0f9ca.png)
when I enter my own array it is not working properly